index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+
+<style type="text/css">
+
+	body {
+
+		background-color: #3CA2E0;     
+	}
+
+	.box {
+
+		width: 40%;
+        	height: 70%;
+        	margin: 100px auto;
+		padding-left: : 50px;
+        	padding-top: 100px;
+        	padding-bottom: 100px;
+        	justify-content: center;
+        	background-color: white;
+        	border-radius: 6px;   
+	}
+	
+	input {
+
+		display: block;
+		border-top: none;
+		border-left: none;
+		border-right: none;
+		text-transform: capitalize;
+		color: #ED90B2;
+		border-color: #B0B4BD;
+		padding:5px;
+	}
+	
+	.buttonA {
+
+		padding: 10px;
+		text-transform: capitalize;
+		background-color: #FF3264;
+		border:none;
+		border-radius: 20px;
+		width: 45%;
+		color: #ED90B2;
+		margin-left: 30%;
+		font-weight: bold;
+		font-size: 15px;		
+	}
+
+	.brandlogo {
+
+		margin-left: 40%;
+	}
+	.tinput {
+
+        	display: flex;
+        	margin-left: 30%;
+        	margin-top: 20px;
+        	margin-bottom: 25px; 
+	}
+</style>
+<head>
+	<title>First Task</title>
+</head>
+<body>
+<div class="box">
+	<form class="form">
+		<img class="brandlogo" src="brand.png">
+		<div class="tinput">
+			<label><img src="username.png"></label>
+	        <input type="text" placeholder="Name">
+		</div>
+		<div class="tinput">
+			<label><img src="password.png"></label>
+		    <input type="password" placeholder="password" >
+		</div>
+		<button type="submit" class="buttonA">Log in</button>

```

Comment:
`buttonA` doesn't specify if this is a login button. Please use expressive variable names.